relay/kusama/src/lib.rs

```diff
@@ -453,11 +454,12 @@ parameter_types! {
 
 impl pallet_transaction_payment::Config for Runtime {
 	type RuntimeEvent = RuntimeEvent;
-	type OnChargeTransaction = FungibleAdapter<Balances, DealWithFees<Self>>;
-	type OperationalFeeMultiplier = OperationalFeeMultiplier;
+	type OnChargeTransaction = FungibleAdapter<Balances, ToAuthor<Self>>;
```

Comment:
Im not in the loop; why was this changed? Maybe one entry in the CHANGELOG please.

Reply:
My bad. I was taking notes from the `rococo` config, but `DealWithFees` uses `ToAuthor` internally, so we shouldn't move this. Reverted back to `DealWithFees`.